### ISSUE TITLE
How to add image label when dockerfile maven build

### ISSUE THREAD

**Issue by dragon9783** [spotify/dockerfile-maven#242]
**Is this a BUG REPORT or FEATURE REQUEST?**:
FEATURE REQUEST

## Description

[Add feature/bug description here]
When build image, how to add image label, like `docker build --label xxxx`

## How to reproduce

[Add steps on how to reproduce this issue]

## What do you expect

[Describe what do you expect to happen]

## What happened instead

[Describe the actual results]

## Software:

- `docker version`: [Add the output of `docker version` here, both client and server]
- Spotify's dockerfile-maven version: [Add dockerfile-maven version here]

## Full backtrace

```text
[Paste full backtrace here]
```

**Comment by mattnworb** [comment]
I don't think this exists at all in the plugin, but we'd be happy to accept any pull requests that add it.

**Comment by thibgc** [comment]
We'd like a feature like that :-)

Currently we work around it by passing build arguments that are then used as the values of LABEL directives in the Dockerfile. (But there is a bug with build arguments containing spaces: #286)
